internal/directory/google/google.go

```diff
@@ -110,6 +110,24 @@ func (p *Provider) UserGroups(ctx context.Context) ([]*directory.Group, []*direc
 		return nil, nil, fmt.Errorf("google: error getting groups: %w", err)
 	}
 
+	userLookup := map[string]apiUserObject{}
```

Comment:
our docs already recommend the `user.readonly` permission, so this worked without having to change my svc account